packages/astro/src/@types/astro.ts

```diff
@@ -1412,34 +1412,51 @@ export interface AstroUserConfig {
 			/**
 			 * @docs
 			 * @name experimental.i18n.fallback
-			 * @type {Record<string, string[]>}
-			 * @version 3.*.*
+			 * @type {Record<string, string>}
+			 * @version 3.5.0
 			 * @description
 			 *
-			 * The fallback system of the locales. By default, the fallback system affect the **content only**, and it doesn't
-			 * do any redirects.
+			 * The content fallback strategy when navigating to pages in non-default languages that do not exist. (e.g. a translated page has not been created).
+			 *
+			 * This a plain object, where a key is a locale that hasn't traslated pages, and the corresponding value is the locale where you want to redirect the missing page to.
+			 *
+			 * #### Examples
+			 *
+			 * In this example, we configure Astro to redirect pages that belong to the `pt` locale to their `es` version, and pages that belong to the `fr` locale to their `en` version.
```

Comment:
```suggestion
			 * The following example configures your content fallback strategy to redirect unavailable pages in `/pt/` to their `es` version, and unavailable pages in `/fr/` to their `en` version. Unavailable `/es/` pages will redirect to the `defaultLocale` (`en`).
```
This is an excellent example! I just tightened up some wording here, but we should also mention what happens for the case that is not explicitly stated. 

Does it default to `en` because it's the `defaultLocale`? Does it give a 404?

Reply:
As for now, we don't redirect to the default locale, and we return a 404

Reply:
Ok, that's easy enough to update! Got it!

Reply:
The above is good to merge as a placeholder for me, but note to self I will check on how we write "return a 404" elsewhere in docs for consistency!